api/http/attachment.go

```diff
@@ -40,21 +40,16 @@ func (service AttachmentListHandler) ServeHTTP(w http.ResponseWriter, r *http.Re
 		return
 	}
 
-	// Get the account information from the data store
+	// Get the account information from the data store.
+	// Proceed even if the account is locked, as files are presented
+	// after application submission, on the Print page.
 	account.ID = id
 	if _, err := account.Get(service.Database, id); err != nil {
 		service.Log.WarnError(api.NoAccount, err, api.LogFields{})
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
-	// If the account is locked then we cannot proceed
-	if account.Locked {
-		service.Log.Warn(api.AccountLocked, api.LogFields{})
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
```

Comment:
Yeah – it is OK. Intent of the account locking is to prevent users from modifying their application data after they've submitted it for review. The 2 calls where I have removed the check (list attachments and get attachment) are read-only in the sense that no applicant form data gets modified.

Locking occurs on `submit()` but currently subsequent success/print page requires the user to be able to make those API calls to download the PDFs.

Medium-term, the whole conops around account locking needs to be revalidated, as it isn't very user friendly. Ideally, the user should have at least several days to come back and print out the archival PDFs of their data and their certification/releases.